### ISSUE TITLE
Upgrading from 1.2 to 2.0.1 gives `globalDefine...has no method 'getCallerModule'`

### ISSUE THREAD

**Issue by joetuson** [alexindigo/node-global-define#3]
I'm getting this error after upgrading from 1.2 to 2.0.1. Any ideas why?

```
    return globalDefine.amdefineWorkaround(globalDefine.getCallerModule());
                                                        ^
TypeError: Object function wrapper(id, deps, initializer) {
    // init module
    define(id, deps, initializer);

    // if module provides id
    // force it to be executed anyway
    if (typeof id == 'string') {
      requiredModule.exports = define.require(id);
    }
  } has no method 'getCallerModule'
   ....
```

**Comment by alexindigo** [comment]
Yep, I forgot to push latest fix. Will do. Thanks. :)

**Comment by alexindigo** [comment]
Fixed in `2.0.2`
